website/src/pages/downloads/windows.tsx

```diff
@@ -9,51 +9,91 @@ import { faMicrosoft, faWindows } from '@fortawesome/free-brands-svg-icons';
 import { faDownload, faPaste, faTerminal } from '@fortawesome/free-solid-svg-icons';
 
 async function grabfilenameforWindows(
-  setDownloadData: React.Dispatch<SetStateAction<{ version: string; binary: string; setup: string }>>,
+  setDownloadData: React.Dispatch<
+    SetStateAction<{
+      version: string;
+      binaryX64: string;
+      binaryArm64: string;
+      setupX64: string;
+      setupArm64: string;
+      airgapsetupX64: string;
+      airgapsetupArm64: string;
+    }>
+  >,
 ): Promise<void> {
   const result = await fetch('https://api.github.com/repos/containers/podman-desktop/releases/latest');
   const jsonContent = await result.json();
   const assets = jsonContent.assets;
-  const windowsSetupAssets = assets.filter(
-    asset => (asset.name as string).endsWith('-setup.exe') && !asset.name.includes('airgap'),
+  const windowsX64SetupAssets = assets.filter(
+    asset => (asset.name as string).endsWith('-setup-x64.exe') && !asset.name.includes('airgap'),
   );
-  if (windowsSetupAssets.length !== 1) {
+  if (windowsX64SetupAssets.length !== 1) {
     throw new Error('Unable to grab setup.exe');
   }
-  const windowsSetupAsset = windowsSetupAssets[0];
+  const windowsArm64SetupAssets = assets.filter(
+    asset => (asset.name as string).endsWith('-setup-arm64.exe') && !asset.name.includes('airgap'),
+  );
+  const setupX64Asset = windowsX64SetupAssets?.[0];
+  const setupX64 = setupX64Asset?.browser_download_url;
+  const setupArm64Asset = windowsArm64SetupAssets?.[0];
+  const setupArm64 = setupArm64Asset?.browser_download_url;
+
+  const binaryOnlyX64WindowsAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('x64.exe') &&
+      !asset.name.includes('airgap') &&
+      asset.name !== setupX64Asset?.name,
+  );
+  const binaryX64 = binaryOnlyX64WindowsAssets?.[0]?.browser_download_url;
 
-  const binaryOnlyWindowsAssets = assets.filter(
+  const binaryOnlyArm64WindowsAssets = assets.filter(
     asset =>
-      (asset.name as string).endsWith('.exe') &&
+      (asset.name as string).endsWith('arm64.exe') &&
       !asset.name.includes('airgap') &&
-      asset.name !== windowsSetupAsset.name,
+      asset.name !== setupX64Asset?.name,
```

Comment:
should it be `setupArm64Asset?.name`?

Reply:
@lstocchi yes looks like a good catch 👍

Reply:
@lstocchi so looking again, it seems to be the expected behaviour

we want to search for an arm64 asset then we want to check it's not the x64 name (so we don't need to say it's not the arm64 asset as we already filtered files ending with arm64.exe) 

I checked again the links using the website preview and they're linking to expected names.